.gitignore

```diff
@@ -0,0 +1,2 @@
+# ignore the test.md file
+test.md
```

Comment:
- [ ] Please add `node_modules` in the `.gitignore` file. As your previous reviewer didn't request this change, I kept it optional this time. But this is actually a mandatory requirement and I think you should follow this for your next projects also.

Reply:
Thank you for the advice @mahbubul14. I certainly will use this valuable information.